docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,172 @@
+
+## Setting up the KinD cluster
+
+```bash
+make kind-extensions-up
+```
+
+This command sets up a new KinD cluster named `gardener-local` and stores the kubeconfig in the `./example/provider-extensions/garden/kubeconfig` file.
+
+> It might be helpful to copy this file to `$HOME/.kube/config` since you will need to target this KinD cluster multiple times.
+Alternatively, make sure to set your `KUBECONFIG` environment variable to `./example/gardener-local/kind-extensions/kubeconfig` for all future steps via `export KUBECONFIG=example/gardener-local/kind-extensions/kubeconfig`.
+
+All following steps assume that you are using this kubeconfig.
+
+Additionally, this command also deploys a local container registry to the cluster as well as a few registry mirrors, that are set up as a pull-through cache for all upstream registries Gardener uses by default.
+This is done to speed up image pulls across local clusters.
+The local registry can be accessed as `localhost:5001` for pushing and pulling.
+The storage directories of the registries are mounted to the host machine under `dev/local-registry`.
+With this, mirrored images don't have to be pulled again after recreating the cluster.
+
+The command also deploys a default [calico](https://github.com/projectcalico/calico) installation as the cluster's CNI implementation with `NetworkPolicy` support (the default `kindnet` CNI doesn't provide `NetworkPolicy` support).
+Furthermore, it deploys the [metrics-server](https://github.com/kubernetes-sigs/metrics-server) in order to support HPA and VPA on the seed cluster.
+
+## Setting up Gardener (Garden on KinD, Seed on Gardener cluster)
+
+```bash
+make gardener-extensions-up
+```
+
+This will first prepare the basic configuration of your KinD and Gardener clusters.
+Afterwards, the images for the Garden cluster are built and deployed into the KinD cluster.
+Finally, the images for the Seed cluster are built, pushed to a container registry on the Seed and the `gardenlet` is started.
+
+## Pause and unpause the KinD cluster
+
+Unlike in the `getting started locally` scenario the KinD cluster can be paused by stopping and keeping its docker container. This is done by:
+
+```bash
+make kind-extensions-down
+```
+
+When you run `make kind-extensions-up` again, you will start the docker container with your previous Gardener configuration again. 
+
+This provides the option to switch off your local KinD cluster fast without leaving orphaned infrastructure elements behind.
+
+
+## Creating a `Shoot` cluster
+
+You can wait for the `Seed` to be ready by running
+
+```bash
+kubectl wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed provider-extensions --timeout=5m
+```
+
+Alternatively, you can run `kubectl get seed provider-extensions` and wait for the `STATUS` to indicate readiness:
```

Comment:
Isn't this done by Skaffold when running `make gardener-extensions-up`?

Reply:
Yes it is. There is a similar paragraph in `getting_started_locally.md` though. 
In this scenario it might make sense to keep this paragraph because there are are other cases which could make the seed `unready` like hibernating the seed or pausing kind. I adapted the command for this use case and added an explanation.